python/ray/rllib/optimizers/sample_batch.py

```diff
@@ -107,7 +111,7 @@ def postprocess_batch_so_far(self):
                 pre_batch, other_batches)
 
         # Append into policy batches and reset
-        for agent_id, post_batch in post_batches.items():
+        for agent_id, post_batch in sorted(post_batches.items()):
```

Comment:
why is sorting needed?

Reply:
It keeps them in a deterministic order for tests.